automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -xe
+
+# This script should be able to execute multus
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-multus-functests.sh
+
+teardown() {
+    make cluster-down
+    rm -rf "${TMP_COMPONENT_PATH}"
+}
+
+function __get-tools() {
+    if [ ! -d bin ]; then
+        mkdir bin
+    fi
+
+    echo "install jq"
+    curl -Lo ./bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+    chmod +x ./bin/jq
```

Comment:
u can use container of jq please

Reply:
jq is used inside the multus source test. changing it to dockerized jq will require changing it in multus repo first and only I can remove this tool installation here, so in any case it shouldn't be in this PR context.